packages/atomic/src/components/search/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -555,7 +554,12 @@ export class AtomicSearchInterface
   };
 
   private async internalInitialization(initEngine: () => void) {
-    await this.commonInterfaceHelper.onInitialization(initEngine);
+    await this.componentWillLoadCalledPromise;
```

Comment:
We first wait for willLoad to be called and completed; otherwise, we won't init the engine with all the fieldsToInclude. Oops.